### DISCUSSION TITLE
Proxy news URLs through Instapaper/Postlight parser w/ plugin

### DISCUSSION THREAD

**Discussion by joestump** [searxng/searxng#4027] [Show and tell]
A while back https://github.com/Hydrophobefireman/reader showed up on r/self-hosted. It's a pretty simple service that uses the Postlight parser to return a nicely formatted news article. 

I created a plugin called [`reader_news_proxy`](https://github.com/joestump/searxng-privacy-plugins) that will automatically proxy URLs based on regex patterns in your `settings.yml`. 

This will automatically redirect various news websites to the Reader proxy for a nicer browsing experience.

**Comment by return42** [comment]
Hey .. thats nice, thanks for sharing :+1: .. if you think https://github.com/joestump/searxng-privacy-plugins/blob/main/reader_news_proxy.py is worth to merge into SearXNG, please open a PR.

**Reply by joestump** [reply]
@return42 I'll take a stab at a more generic version of this. 👍🏼
